general/virtuoso-features-matrix.ttl

```diff
@@ -0,0 +1,237 @@
+ns15:this
+	schema:name	"Meta Schema language" , "Meta Schema Language" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns16:this
+	schema:name	"Network based Query Parallelization" , "Network-based Query Parallelization" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns17:this
+	schema:name	"Parallelized SQL Query Execution" , "Parallelized Execution of SQL Queries" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns18:this
+	schema:name	"RDF Data Management" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns19:this
+	schema:name	"RDF Middleware" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns20:this
+	schema:name	"RDF Views over External Data Sources" , "Linked Data Views over External Data Sources" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns21:this
+	schema:name	"RDF Views over Native SQL Data Sources" , "Linked Data Views over Native SQL Data Sources" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns22:this
+	schema:name	"RDF aware Clustering" ;
+	feat:isFeatureOf	ns3:this , ns4:this ;
+	rdf:type	feat:ProductFeature .
+ns23:this
+	schema:name	"RDF based Linked Data Deployment" , "RDF-based Linked Data Deployment" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns24:this
+	schema:name	"Runtime Hosting" , "Run Time Hosting" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns25:this
+	schema:name	"SPARQL Query Language" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns26:this
+	schema:name	"Storage Engine" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns27:this
+	schema:name	"Supports Broad Range of Data Access Standards" , "Supports a Broad Range of Data Access Standards" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns28:this
+	schema:name	"WebDAV Compliant Content Manager" , "WebDAV-Compliant Content Manager" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns29:this
+	schema:name	"Web Service Deployment Platform" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns30:this
+	schema:name	"Web Services to RDF Linked Data" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns32:this
+	schema:name	"Attribute-based Access Controls"^^dt:string ;
+	feat:isFeatureOf	ns33:this ;
+	rdf:type	feat:ProductFeature .
+ns34:this
+	schema:name	"Custom Inference Rules"^^dt:string ;
+	feat:isFeatureOf	ns33:this ;
+	rdf:type	feat:ProductFeature .
+ns35:this
+	schema:name	"Enhanced Relational Mapping"^^dt:string , "Enhanced Relational Mapping" ;
```

Comment:
`^^dt:string` is rarely helpful. `@en` is likely to be, when we get around to internationalizing the website.
```suggestion
   schema:name      "Attribute-based Access Controls"@en ;
   feat:isFeatureOf ns33:this ;
   rdf:type         feat:ProductFeature .
ns34:this
   schema:name      "Custom Inference Rules"@en ;
   feat:isFeatureOf ns33:this ;
   rdf:type         feat:ProductFeature .
ns35:this
   schema:name      "Enhanced Relational Mapping"@en ;
```